SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetDriver.cpp

```diff
@@ -751,6 +751,12 @@ void USpatialNetDriver::SpatialProcessServerTravel(const FString& URL, bool bAbs
 		return;
 	}
 
+	if (NetDriver->LoadBalanceStrategy->GetMinimumRequiredWorkers() > 1)
+	{
+		UE_LOG(LogGameMode, Warning, TEXT("Server travel is not supported on a deployment with multiple workers."));
```

Comment:
Would make error, failed server travel sounds fatal for a deployment

Reply:
Good point.